### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by clochix**
This should cleanup the application error log by removing deprecation warnings.

### CODE REVIEW

File: server/models/note.coffee [hunk 1]
```diff
@@ -117,28 +120,12 @@ Note::updatePath = (newPath, callback) ->
...
-# TMP VERSION - to be replaced with above
-Note::destroyWithChildren = (callback) ->
-
-    oldPath = @path.slice(0)
-    query =
-        startkey: oldPath # [a, b, oldtitle]
-        endkey:   oldPath.concat [{}] # [a, b, oldtitle,{}]
-
-    Note.request "path", query, (err, notes) ->
+   oldPath = @path.slice(0)
```
Comment (frankrousseau) [inline server/models/note.coffee]: Indentation is wrong here.

File: package.json [hunk 2]
```diff
@@ -11,12 +11,12 @@
   "main": "server.js",
   "dependencies": {
     "async": "*",
-    "americano": "=0.3.3",
-    "americano-cozy": "=0.2.2",
-    "cozy-realtime-adapter": "=0.12.0"
+    "americano": "=0.4.2",
```
Comment (nono) [inline package.json]: @clochix IIRC, you have to also update server.coffee if you upgrade americano. The callback for start takes one more argument - err - that comes first. See https://github.com/cozy/americano#handmade for an example

Comment (frankrousseau) [inline package.json]: To keep the Cozy repositories consistent, the guideline to write dependency version is to not use the `=`.  It has the same effect and unclutters a little bit the manifest.

**Comment by nono** [comment]
@clochix any news on this PR?

**Comment by clochix** [comment]
Thanks for noticing @nono, I just fixed the callback parameters.

**Pull request merged**